db/migrations/2025_03_11_113040_create_submissions_and_excel_files.py

```diff
@@ -0,0 +1,45 @@
+import sqlalchemy as sql
+
+
+def up(conn):
+    query = """
+        CREATE TABLE ExcelFiles (
+            id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+
+            filename VARCHAR(255),
+            size     INT NOT NULL,
+            content  LONGBLOB NOT NULL,
+
+            createdAt DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP
+        );
+    """
+    conn.execute(sql.text(query))
+
+    query = """
+        CREATE TABLE Submissions (
+            id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+
+            projectUniqueID VARCHAR(100) COLLATE utf8mb4_bin,
+            studyUniqueID   VARCHAR(100) COLLATE utf8mb4_bin,
+            userUniqueID    VARCHAR(100) COLLATE utf8mb4_bin,
+
+            studyDesign JSON DEFAULT (JSON_OBJECT()),
```

Comment:
Could you document a bit this in terms of what we store as json?
I get it but I think it's good to have something concrete for the future.